src/Raven.Server/Web/Studio/StudioIndexHandler.cs

```diff
@@ -112,6 +113,58 @@ public async Task PostIndexFields()
 
             return result;
         }
+
+        private static HashSet<AdditionalAssembly> ConvertToAdditionalAssemblies(BlittableJsonReaderArray jsonArray)
+        {
+            if (jsonArray == null || jsonArray.Length == 0)
+                return null;
+
+            var result = new HashSet<AdditionalAssembly>();
+           
+            foreach (BlittableJsonReaderObject assemblyJson in jsonArray)
+            {
+                var assembly = GetAssembly(assemblyJson);
+                if (assembly != null)
+                {
+                    result.Add(assembly);
+                }
+            }
+
+            return result;
+        }
+
+        private static AdditionalAssembly GetAssembly(BlittableJsonReaderObject json)
+        {
+            json.TryGet(nameof(AdditionalAssembly.AssemblyName), out string assemblyName);
+            json.TryGet(nameof(AdditionalAssembly.AssemblyPath), out string assemblyPath);
+            json.TryGet(nameof(AdditionalAssembly.PackageName), out string packageName);
+            json.TryGet(nameof(AdditionalAssembly.PackageVersion), out string packageVersion);
+            json.TryGet(nameof(AdditionalAssembly.PackageSourceUrl), out string packageSourceUrl);
+                
+            json.TryGet(nameof(AdditionalAssembly.Usings), out BlittableJsonReaderArray usingsArray);
```

Comment:
usingsArray can be null. You need to wrap the code below in `if`

Reply:
done